src/main.rs

```diff
@@ -50,6 +50,7 @@ cfg_if! {
                             <!DOCTYPE html>
                             <html>
                               <head>
+                                <meta charset="utf-8" />
```

Comment:
This fixes a bug where there will be a flash of incorrect text when first loading the page.